index.js

```diff
@@ -36,6 +36,10 @@ function mapApiGatewayEventToHttpRequest(event, context, socketPath) {
     const eventWithoutBody = Object.assign({}, event)
     delete eventWithoutBody.body
 
+    // NOTE: API Gateway is not setting Content-Length header on any requests even when they have a body  
```

Comment:
I don't think this is true. It correctly sets it for POST and PUT, right? Otherwise we'd be seeing this error on every method. Confirmed methods where it doesn't appear are GET and DELETE and I wouldn't be surprised if OPTIONS also didn't have the header.